### DISCUSSION TITLE
Centering content in reveal presentations

### DISCUSSION THREAD

**Discussion by neeldhara** [quarto-dev/quarto-cli#3608] [Q&A] [answered]
I'm trying to recreate [this presentation](https://slides.com/neeldhara/daiict) (also made in reveal but with the help of the slides.com UI) in Quarto. Any suggestions how how I can get the main content to be centre-aligned (vertically _and_ horizontally), like it is in [this slide](https://slides.com/neeldhara/daiict#/0/3)?

**Comment by mcanouil** [comment, marked as answer]
Depending on what you want to be centered: #1386 and/or #2951 might be of interest.

**Reply by neeldhara** [reply]
Sorry I forgot to acknowledge this: thank you for the pointers!
